pkg/nirmata/gke_clusterType.go

```diff
@@ -72,6 +72,41 @@ func resourceGkeClusterType() *schema.Resource {
 					return
 				},
 			},
+			"location_type": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"node_locations": {
+				Type: schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Required: true,
+			},
+			"enable_secrets_encryption": {
+				Type:     schema.TypeBool,
+				Required: true,
```

Comment:
can we default to `true`

Reply:
IF we make default true, then it's mandatory to add **Secret Encryption Key**. and this value user have to specified.
So we kept default false.

Reply:
ok